src/main/java/frc/trigon/robot/subsystems/climber/ClimberVisualization.java

```diff
@@ -0,0 +1,144 @@
+package frc.trigon.robot.subsystems.climber;
+
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.Logger;
+import org.trigon.utilities.Conversions;
+
+/**
+ * A 2D representation of the climber mechanism.
+ */
```

Comment:
This is also a 3D representation. Explain why this needs a whole class, and everything it does.